### PR TITLE
Updated ChangeLog.md

### PR DESCRIPTION
**Pull request by Subhashvysyaraju**
## Description

Updated Change Log.Md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+#### Android
+  
+* **[Feature]** Allow users to set userId that applies to crashes, error and push logs. This feature adds an API, but is not yet supported on the App Center backend.
+* **[Fix]** Do not delete old logs when trying to add a log larger than the maximum storage capacity.
+* **[Fix]** Fix error detection of setMaxStorageSize API if database uses custom page size.
+* **[Fix]** Fix minimum storage size verification to match minimum possible value.
+* **[Fix]** Fix disabling logging of network state changes according to AppCenter.getLogLevel.
+* **[Fix]** Fix logs duplication on unstable network.
+
+#### IOS
```
Comment (guperrot) [inline CHANGELOG.md]: IOS => iOS

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
 # App Center SDK for .NET Change Log
 
-## Version 1.12.0 (Under active development)
+## Version 1.12.0 
+
+### AppCenter
+
+#### Android
+  
+* **[Feature]** Allow users to set userId that applies to crashes, error and push logs. This feature adds an API, but is not yet supported on the App Center backend.
```
Comment (guperrot) [inline CHANGELOG.md]: UserId feature is not exposed in Xamarin, please remove that entry.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,27 @@
...
+* **[Fix]** Do not delete old logs when trying to add a log larger than the maximum storage capacity.
+* **[Fix]** Fix error detection of setMaxStorageSize API if database uses custom page size.
```
Comment (guperrot) [inline CHANGELOG.md]: This API does not exist on Xamarin, please also remove that entry.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,27 @@
...
+* **[Fix]** Fix minimum storage size verification to match minimum possible value.
```
Comment (guperrot) [inline CHANGELOG.md]: Same please remove that one

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,27 @@
...
+* **[Fix]** Fix disabling logging of network state changes according to AppCenter.getLogLevel.
+* **[Fix]** Fix logs duplication on unstable network.
+
+#### IOS
+
+* **[Feature]** Allow users to set userId that applies to crashes, error and push logs. This feature adds an API, but is not yet supported on the App Center backend.
```
Comment (guperrot) [inline CHANGELOG.md]: remove this line

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,27 @@
...
+* **[Fix]** Do not delete old logs when trying to add a log larger than the maximum storage capacity.
+* **[Fix]** Fix minimum storage size verification to match minimum possible value.
```
Comment (guperrot) [inline CHANGELOG.md]: remove this line

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,27 @@
...
+* **[Fix]** Fix reporting carrier information using new iOS 12 APIs when running on iOS 12+.
+* **[Fix]** Fix a memory leak issue during executing SQL queries.
+* **[Fix]** Fix a keychain permission issue on macOS applications.
+* **[Feature]** Add preview support for arm64e CPU architecture.
```
Comment (guperrot) [inline CHANGELOG.md]: remove this line

File: CHANGELOG.md [hunk 8]
```diff
@@ -9,6 +30,35 @@
 * **[Fix]** Downgrade the platforms verification error to a warning.
```
Comment (guperrot) [inline CHANGELOG.md]: Add `#### UWP` (blank line after)

File: CHANGELOG.md [hunk 9]
```diff
@@ -9,6 +30,35 @@
...
 * **[Fix]** Update vulnerable `Microsoft.NETCore.UniversalWindowsPlatform` dependency from version `5.2.2` to `5.2.6`.
 
+#### ANDROID
```
Comment (guperrot) [inline CHANGELOG.md]: ANDROID => Android

File: CHANGELOG.md [hunk 10]
```diff
@@ -9,6 +30,35 @@
...
+
+* **[Fix]** Fix a bug where crash data file could leak when the database is full.
+
+#### IOS
```
Comment (guperrot) [inline CHANGELOG.md]: Please remove this section, this is for native ios

File: CHANGELOG.md [hunk 11]
```diff
@@ -9,6 +30,35 @@
...
+
+* **[Feature]** Add preview support for arm64e CPU architecture by using PLCrashReporter 1.2.3-rc1. PLCrashReporter 1.2.3-rc1 is a fork of the official repository and can be found at https://github.com/Microsoft/PLCrashReporter. It is based on version It is maintained by the App Center team and based on PLCrashReporter 1.2.1 (commit fda23306) with additional fixes and changes.
+
+### AppCenterAnalytics
+
+#### IOS
```
Comment (guperrot) [inline CHANGELOG.md]: Remove the iOS section here as this is a native only change

File: CHANGELOG.md [hunk 12]
```diff
@@ -9,6 +30,35 @@
...
+
+* **[Feature]** Add preview support for arm64e CPU architecture.
+
+### AppCenterDistribute
+
+#### IOS
```
Comment (guperrot) [inline CHANGELOG.md]: remove the ios section here too

File: CHANGELOG.md [hunk 13]
```diff
@@ -9,6 +30,35 @@
...
+
+* **[Feature]** Add preview support for arm64e CPU architecture.
+
+### AppCenterPush
+
+#### IOS
```
Comment (guperrot) [inline CHANGELOG.md]: remove the ios section here too

File: CHANGELOG.md [hunk 14]
```diff
@@ -9,6 +30,35 @@
...
+
+* **[Feature]**  Add preview support for arm64e CPU architecture.
+
+#### ANDROID
```
Comment (guperrot) [inline CHANGELOG.md]: ANDROID => Android

File: CHANGELOG.md [hunk 15]
```diff
@@ -9,6 +30,35 @@
...
+
+ * **[Fix]** Fix push foreground listener after re-enabling push service.
```
Comment (guperrot) [inline CHANGELOG.md]: On Xamarin this bug fix needs to be documented differently:

**[Fix]** Fix `PushNotificationReceived` event for pushes received in foreground after re-enabling the push service.

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,14 +1,44 @@
 # App Center SDK for .NET Change Log
 
-## Version 1.12.0 (Under active development)
+## Version 1.12.0 
+
+### AppCenter
+
+#### Android
```
Comment (guperrot) [inline CHANGELOG.md]: Blank line after a section

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,14 +1,44 @@
...
+* **[Fix]** Do not delete old logs when trying to add a log larger than the maximum storage capacity.
+* **[Fix]** Fix minimum storage size verification to match minimum possible value.
+* **[Fix]** Fix disabling logging of network state changes according to AppCenter.getLogLevel.
+* **[Fix]** Fix logs duplication on unstable network.
+
+#### iOS
```
Comment (guperrot) [inline CHANGELOG.md]: Blank line after a section

File: CHANGELOG.md (hunk 7 above)
Comment (guperrot) [reply CHANGELOG.md]: this was not updated, this line must be removed

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,14 +1,44 @@
...
+* **[Fix]** Do not delete old logs when trying to add a log larger than the maximum storage capacity.
+* **[Fix]** Fix minimum storage size verification to match minimum possible value.
+* **[Fix]** Fix reporting carrier information using new iOS 12 APIs when running on iOS 12+.
+* **[Fix]** Fix a memory leak issue during executing SQL queries.
+* **[Fix]** Fix a keychain permission issue on macOS applications.
```
Comment (guperrot) [inline CHANGELOG.md]: Remove this line, we don't support MacOS on Xamarin

File: CHANGELOG.md [hunk 19]
```diff
@@ -1,14 +1,44 @@
...
+* **[Feature]** Add preview support for arm64e CPU architecture.
 
 ### AppCenterCrashes
 
 #### UWP
```
Comment (guperrot) [inline CHANGELOG.md]: Blank line after a section

File: CHANGELOG.md [hunk 20]
```diff
@@ -1,14 +1,44 @@
...
-
 * **[Fix]** Downgrade the platforms verification error to a warning.
 * **[Fix]** Update vulnerable `Microsoft.NETCore.UniversalWindowsPlatform` dependency from version `5.2.2` to `5.2.6`.
 
+#### Android
```
Comment (guperrot) [inline CHANGELOG.md]: Blank line after a section

File: CHANGELOG.md (hunk 12 above)
Comment (guperrot) [reply CHANGELOG.md]: Need to remove this section, comment was resolved without fixing

File: CHANGELOG.md (hunk 13 above)
Comment (guperrot) [reply CHANGELOG.md]: Need to remove this section, comment was resolved without fixing

File: CHANGELOG.md [hunk 21]
```diff
@@ -1,14 +1,44 @@
...
+
+#### IOS
+* **[Feature]** Add preview support for arm64e CPU architecture.
+
+### AppCenterPush
+
+#### IOS
+* **[Feature]**  Add preview support for arm64e CPU architecture.
+
+#### Android
```
Comment (guperrot) [inline CHANGELOG.md]: Blank line after a section

File: CHANGELOG.md [hunk 22]
```diff
@@ -1,14 +1,44 @@
...
+* **[Fix]** Fix PushNotificationReceived event for pushes received in foreground after re-enabling the push service
```
Comment (guperrot) [inline CHANGELOG.md]: PushNotificationReceived => `PushNotificationReceived`

Comment (guperrot) [reply CHANGELOG.md]: dot at the end.

File: CHANGELOG.md (hunk 18 above)
Comment (Subhashvysyaraju) [reply CHANGELOG.md]: this was not updated, this line must be removed . i didnt get this . here what exactly need to changed

**Pull request merged**